### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by pudelkoM**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,15 +27,19 @@ again.
...
 
-4. Wait for [CI checks](https://circleci.com/gh/stratum/stratum) to pass. You can check the [coverage report](https://codecov.io/gh/stratum/stratum) after they ran. Repeat steps 1. and 2. as necessary. **Passing CI is mandatory.**
+4. Check code style compliance with `cpplint` and `clang-format` (pre-installed in development docker container).
 
-5. Request review. You can add anyone you think is helpful, but only Collaborators and above can give final approval. **All changes must get at least one approval**. Join one of the [communication channels](https://wiki.opennetworking.org/display/COM/Stratum+Wiki+Home+Page) to bring additional attention to your PR.
+5. Create a [PR on Github](https://github.com/stratum/stratum/compare). Consider [allowing maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to make changes if you want direct assistance from Contributers.
```
Comment (Yi-Tseng) [inline CONTRIBUTING.md]: if you want direct assistance from **_maintainers_**. ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -19,23 +19,31 @@ again.
 
 ### General Information
 
- - One PR for one feature or change request. Prefer multiple smaller PRs to one gigantic one. Multiple commits during development are OK; see next point.
+Stratum follows [Google's Engineering Practices](https://google.github.io/eng-practices/review/developer/). Use this document as a guide when submitting code.
+
+Some additional points:
+ 
+ - Submit you changes early and often. GitHub has (Draft PRs)[https://github.blog/2019-02-14-introducing-draft-pull-requests/] that allow you to share your code with others during development. Input and corrections early in the process prevent huge changes later.
```
Comment (Yi-Tseng) [inline CONTRIBUTING.md]: "Submit `your` changes early and often" ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -19,23 +19,31 @@ again.
...
+ - Consider opening a separate issue describing the technical details there and [link it to the PR](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords). This keeps code review and design discussions clean.
 
 ### Steps to Follow
 
-1. Make sure your code passes unit tests. Create new ones for new code and **add them to the [build-targets.txt](.circleci/build-targets.txt) and [test-targets.txt](.circleci/test-targets.txt) files** in the same PR, so CI can pick them up! You can also run them locally with bazel: `xargs -a .circleci/test-targets.txt bazel test`
+1. Fork Stratum into your namespace via the fork button on GitHub.
+
+2. Make your code changes.
+
+3. Pass all unit tests locally. Create new tests for new code and **add the targets to the [build-targets.txt](.circleci/build-targets.txt) and [test-targets.txt](.circleci/test-targets.txt) files** in the same PR, so CI can pick them up! To run all currently enabled tests execute this in the Stratum root directory: `xargs -a .circleci/test-targets.txt bazel test`
```
Comment (Yi-Tseng) [inline CONTRIBUTING.md]: Maybe `Execute the following command in the Stratum root directory to run all currently enabled tests` ?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -19,23 +19,31 @@ again.
...
 
-2. Check code style with `cpplint` and `clang-format` (pre-installed in dev docker image).
+4. Check code style compliance with `cpplint` and `clang-format` (pre-installed in development docker container).
 
-3. Create a [PR on Github](https://github.com/stratum/stratum/compare). Either directly on the stratum repo or in your own fork. (Consider [allowing maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to make changes)
+5. Create a [PR on Github](https://github.com/stratum/stratum/compare). Consider [allowing maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to make changes if you want direct assistance from maintainers.
```
Comment (Yi-Tseng) [inline CONTRIBUTING.md]: `Create a [Pull Request]` ...

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -19,23 +19,31 @@ again.
...
  
- - Describe what your PR solves and why. Consider opening a separate issue describing the technical details there and [link it to the PR](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords). This keeps code review and design discussions clean.
+ - Consider opening a separate issue describing the technical details there and [link it to the PR](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords). This keeps code review and design discussions clean.
 
 ### Steps to Follow
 
-1. Make sure your code passes unit tests. Create new ones for new code and **add them to the [build-targets.txt](.circleci/build-targets.txt) and [test-targets.txt](.circleci/test-targets.txt) files** in the same PR, so CI can pick them up! You can also run them locally with bazel: `xargs -a .circleci/test-targets.txt bazel test`
+1. Fork Stratum into your namespace via the fork button on GitHub.
```
Comment (bocon13) [inline CONTRIBUTING.md]: your namespace => your personal or organization account?

**Pull request merged**